messagix/http.go

```diff
@@ -190,7 +190,7 @@ func (c *Client) makeRequestDirect(url string, method string, headers http.Heade
 		}
 	}()
 	if err != nil {
-		c.UpdateProxy(fmt.Sprintf("http request error: %v", err.Error()))
+		c.UpdateProxy(fmt.Sprintf("http request error: %w", err.Error()))
```

Comment:
This one is still wrong:

```suggestion
		c.UpdateProxy(fmt.Sprintf("http request error: %s", err.Error()))
```